common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -59,17 +68,57 @@ def _get_capa_types():
     ], key=lambda item: item.get('display_name'))
 
 
-class LibraryContentFields(object):
+#pylint: disable=abstract-method
```

Comment:
> Hmm, I don't see any other comment by you on this PR?

Whoops, I must have forgotten to save the comment before submitting.

Anyway, I believe that this #pylint-disable will apply to the entire file. If you put it within the class, then it'll limit its scope. I know that it's coming from the existing code, so up to you whether this seems worth changing or not.

@symbolist @ormsbee  In lieu of any objections, my plan is to merge Monday morning (EDT).